### PR TITLE
Add Groupware section

### LINKED ISSUE #1866: Document calendar invitations
- [x] Overall invitation process of calendar should be documented.
- [x] FreeBusy
- [x] Big note that invitations have to be sent from your own calendar, not a shared one.

### PR DESCRIPTION
**Pull request by georgehrke**
Closes #1866

### CODE REVIEW

**Comment by tcitworld** [comment]
Not finished, working on https://github.com/nextcloud/documentation/issues/1866 as well

**Comment by skjnldsv** [comment]
> Not finished, working on #1866 as well

Consider this as an advance on your future reviews! :grin:

File: admin_manual/groupware/calendar.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+See :doc:`../configuration_server/occ_command` section Dav commands.
+
+You'll also need to change the sending mode from ``background-job`` to ``occ``::
+
+ php occ config:app:set dav sendEventRemindersMode --value occ
+
+If you don't use this dedicated command, the reminders will just be send as soon as possible when the background jobs run.
+
+FreeBusy
```
Comment (tcitworld) [inline admin_manual/groupware/calendar.rst:48]: What did you want to write about FreeBusy for admins @georgehrke ?

Comment (tcitworld) [reply admin_manual/groupware/calendar.rst:48]: ping @georgehrke, otherwise I'll like to merge this

Comment (georgehrke) [reply admin_manual/groupware/calendar.rst:48]: I think i wanted to document the `disableFreeBusy` option.

You can set `php occ config:app:set dav disableFreeBusy --value yes` to disable FreeBusy. On certain, especially very big instances, you don't necessarily want everyone to be able to see all FreeBusy data.

@tcitworld Would you mind adding that? :)

Comment (tcitworld) [reply admin_manual/groupware/calendar.rst:48]: Yup, will do.

Comment (tcitworld) [reply admin_manual/groupware/calendar.rst:48]: @georgehrke Done

**Comment by tcitworld** [comment]
It should be applicable to stable17 as well.

**Pull request merged**